packages/components/src/table/table.js

```diff
@@ -118,9 +133,10 @@ class Table extends Component {
 			rowHeader,
 			rows,
 		} = this.props;
-		const { isScrollable, tabIndex } = this.state;
+		const { isScrollable, isScrollableBack, tabIndex } = this.state;
 		const classes = classnames( 'woocommerce-table__table', classNames, {
 			'is-scrollable': isScrollable,
+			'is-scrollable-back': isScrollableBack,
```

Comment:
Nitpicky, but this might be easier to understand if we change these to `isScrollableLeft` and `isScrollableRight`.  What do you think?

Reply:
Definitely agreed – done in https://github.com/woocommerce/woocommerce-admin/pull/4179/commits/6b8340e370668309dcbfccf393f28b2096730f37